Tehtris/tehtris-edr/tests/test_policy_alert.json

```diff
@@ -102,6 +102,13 @@
         "name": "User Execution: Web requests from unusual sources"
       }
     },
+    "url": {
+      "domain": "astgedgecp.region.local",
+      "original": "http://astgedgecp.region.local",
+      "port": 80,
+      "scheme": "http",
+      "subdomain": "astgedgecp.region"
+    },
```

Comment:
Due to the extraction of
https://github.com/SEKOIA-IO/intake-formats/blob/08aca05c530a7789777310d866d35d2536bfaddb/Tehtris/tehtris-edr/tests/test_policy_alert.json#L48